src/library-authoring/section-subsections/LibraryContainerChildren.tsx

```diff
@@ -57,6 +60,18 @@ const ContainerRow = ({ containerKey, container, readOnly }: ContainerRowProps)
     }
   };
 
+  /* istanbul ignore next */
+  const scheduleJumpToTags = useRunOnNextRender(() => {
+    // TODO: Ugly hack to make sure sidebar shows manage tags section
+    // This needs to run after all changes to url takes place to avoid conflicts.
+    setTimeout(() => setSidebarAction(SidebarActions.JumpToManageTags), 250);
+  });
```

Comment:
Wow, I wish there were a better way to do this. I see we're already using it in a few other places.

What is the reason we can't just do something like

```ts
navigateTo({ selectedItemId: container.originalId, sidebarAction: SidebarActions.JumpToManageTags });
```
?

Reply:
I am working on this in https://github.com/open-craft/frontend-app-authoring/pull/91